### ISSUE TITLE
Drop No finding classes , but loss is not decreasing 

### ISSUE THREAD

**Issue by Mahmood1968** [dell-ai-engineering/BigDL-ImageProcessing-Examples#12]
I have dropped No finding classes which make the data set size around 51000 images . I run the code for  10 epochs , 16 batch size and learning rate ; 0.001 and decay rate : 0.0001 . Still the loss function error is not decreasing in the right way . Any Suggestion 
![drop_no_finding](https://user-images.githubusercontent.com/24418838/47452455-f37ab300-d78f-11e8-9060-8137ba6f1cfb.png)

**Comment by Mahmood1968** [comment]
If you look at the error is highly fluctuated and not decreasing, what does this mean ? However,  I am still waiting to get the final AUC .
